webapp/packages/core-dialogs/src/CommonDialog/DialogsPortal.tsx

```diff
@@ -82,11 +82,20 @@ function NestedDialog({
 
   const DialogComponent = dialog.component;
 
+  const backdropClickHandler = (e: React.MouseEvent<HTMLDivElement, MouseEvent>) => {
+    if (e.currentTarget.isEqualNode(e.target as HTMLElement)) {
```

Comment:
`if (e.currentTarget.isEqualNode(e.target as HTMLElement) && !dialog.options?.persistent)`
Dialogue can be closed only when it's not persistent